hbw/lib/hbw/config.rb

```diff
@@ -17,5 +17,9 @@ def [](*keys)
         result
       end
     end
+
+    def set(key, value)
+      @value[key] = value
+    end
```

Comment:
Don't like it, but it seems we can't avoid this or smth like this in the e2e test.
Just for me, it's better to stub the container with config, but in general, it's almost the same. Would be great to write some units with dependency substitution, but I'm not sure the current code is ready for it.

Reply:
So, we leave it as it is?

Reply:
Yeah. It's not about this ticket, but one time we should start using DI at least for things like config or HTTP-client and write units if we need simple checks instead of heavy e2e.